### ISSUE TITLE
Clarify analysis names "IO" and "Input/Output"

### ISSUE THREAD

**Issue by ebugden** [theia-ide/theia-trace-extension#275]
Rename "IO" and "Input/Output" analyses to clarify what they do and what the difference between them is.

**Comment by ebugden** [comment]
Analysis information (from @tahini):

> * **Input/Output** is related to disk (block devices)
> * **IO** is more towards the open/read/write/etc syscalls (includes sockets), linked to a thread/process
>
> Probably Input/Output should be renamed

**Comment by bhufmann** [comment]
The trace server is providing the name to display and the change needs to be done there and not in the front-end. The front-end just shows what is defined in the server.

**Comment by ebugden** [comment]
So you mean this issue is in the wrong repository?

**Comment by tahini** [comment]
I just added the Trace Server label to the issue. We use this issue tracker to also track stuff that regards the trace extension, but for some other repos as well.

**Comment by ebugden** [comment]
So analysis naming issues can stay here (even if the trace server code isn't here)?

**Comment by bhufmann** [comment]
yes, we can track it here for this trace viewer since it's the user facing part. It was just an fyi. Adding the Trace Viewer label will help to know where it needs to be fixed.
